src/components/MatPassContent.tsx

```
@@ -177,43 +253,108 @@ const CartItemComponent = ({
+          <button
+            onClick={handleQuantityIncrease}
+            className={`h-7 w-7 flex items-center justify-center ${
+              isAcro
+                ? "text-purple-600 hover:bg-purple-50"
+                : "text-blue-600 hover:bg-blue-50"
+            } transition-colors`}
+            aria-label={
+              language === "en" ? "Increase quantity" : "Aumentar cantidad"
+            }
+          >
+            <Plus size={14} />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)